config/minter.sandbox.json

```diff
@@ -1,10 +1,10 @@
 {
 	"rpc": "http://localhost:20000",
 	"admin": {
-		"address": "tz1YPSCGWXwBdTncK2aCctSZAXWvGsGwVJqU",
-		"secret": "edsk3RFgDiCt7tWB2oe96w1eRw72iYiiqZPLu9nnEY23MYRp2d8Kkx"
+		"address": "tz1VSUr8wwNhLAzempoch5d6hLRiTh8Cjcjb",
+		"secret": "edskRpm2mUhvoUjHjXgMoDRxMKhtKfww1ixmWiHCWhHuMEEbGzdnz8Ks4vgarKDtxok7HmrEo1JzkXkdkvyw7Rtw6BNtSd7MJ7"
 	},
 	"contracts": {
-		"nft": "KT1HpTpGoktbMf9BFg5kgaCQR7uGw2AVabm5"
+		"nft": "KT1FVT95brPwbuWb6epuqU95hjzwyz2oqdz1"
 	}
 }
```

Comment:
@emishur This is working well overall, but we likely need some way to configure the admin keys and contracts separately so we can omit them from source control while providing something like `config/minter.example.json` as a reference